client/packages/invoices/src/Returns/modals/CustomerReturn/CustomerReturn.tsx

```diff
@@ -78,8 +80,9 @@ export const CustomerReturnEditModal = ({
 
   const onOk = async () => {
     try {
-      !isDisabled && (await save());
+      let customerReturn = !isDisabled && (await save());
       onCreate?.();
+      !!customerReturn && customerReturn?.originalShipment?.id && isNewReturn && success(t('messages.customer-return-created-verified'))();
```

Comment:
Type mapping was a little bit messy to make this work where it could be false | invoice node

Reply:
Do we need to check original shipment id and is new return? If api returns with a result (!!customerReturn) then it would prompt the message, correct?